packages/shorebird_code_push/README.md

```diff
@@ -0,0 +1,5 @@
+# Shorebird Code Push
+
+[![License: MIT][license_badge]][license_link]
```

Comment:
Looks like the links are missing here (license_badge and license_link)

Reply:
Fixed.